build.rs

```diff
@@ -130,8 +130,9 @@ fn run<S: Into<String> + AsRef<std::ffi::OsStr>>(
     cmd.stdout(process::Stdio::piped());
     cmd.stderr(process::Stdio::piped());
 
-    if let Some(dir) = dir {
-        cmd.current_dir(dir.into());
+    let dir = dir.map(Into::into);
```

Comment:
All of the changes in this file are not strictly necessary for the fix. I just did some refactoring as I was trying to debug the issue.